src/main/kotlin/com/exactpro/th2/httpserver/server/Th2HttpServer.kt

```diff
@@ -90,6 +91,12 @@ internal class Th2HttpServer(
         var isClosing = false
         while (!isClosing) {
             runCatching {
+                if(!client.isConnected || client.isClosed || client.isInputShutdown) {
+                    if(reportClientClosingConnection) {
```

Comment:
This flag looks like redundant if this case doesn't reproduce in gracefully handling

Reply:
Client can close connection and it will be graceful from protocol perspective.

https://www.w3.org/Protocols/HTTP/1.1/draft-ietf-http-v11-spec-01#Connection
`The persistent connection ends when either side closes the connection or after the receipt of a response which lacks the "keep-alive" keyword. `

Reply:
OK, but our server doesn't complete its function and user should know that it is happened.
I think conn should emit event when user close connection before complete request